korman/properties/modifiers/gui.py

```diff
@@ -482,7 +482,7 @@ def pre_export(self, exporter, bo):
         else:
             rgn_obj = self.clickable_region
 
-        if self.shareable:
+        if self.shareable and self.link_type == "kOriginalBook":
```

Comment:
To reduce the amount of potentially duplicated logic, Korman usually has Python properties for this. Eg, in the class:

```python
    @property
    def export_share_region(self) -> bool:
        return self.shareable and self.link_type == "kOriginalBook"
```

Then this line becomes:
```suggestion
        if self.export_share_region:
```